### PR TITLE
Resolve puppet-lint notices

### PR DESCRIPTION
**Pull request by jcpunk**
#### Pull Request (PR) description
puppet-lint flagged these lines

### CODE REVIEW

File: types/date.pp [hunk 1]
```diff
@@ -1,10 +1,9 @@
 # Valid $date (day of month) parameter to Cron::Job.
 type Cron::Date = Variant[
   Integer[1,31],
-  Pattern[/(?x)\A(
+Pattern[/(?x)\A(
```
Comment (igalic) [inline types/date.pp:4]: why is this indentation gone?

File: types/minute.pp [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Valid $minute parameter to Cron::Job.
 type Cron::Minute = Variant[
   Integer[0,59],
-  Pattern[/(?x)\A(
+Pattern[/(?x)\A(
```
Comment (igalic) [inline types/minute.pp:4]: why is this indentation gone?

File: types/hour.pp [hunk 3]
```diff
@@ -1,10 +1,9 @@
 # Valid $hour parameter to Cron::Job.
 type Cron::Hour = Variant[
   Integer[0,23],
-  Pattern[/(?x)\A(
+Pattern[/(?x)\A(
```
Comment (igalic) [inline types/hour.pp:4]: why is this indentation gone?

File: types/month.pp [hunk 4]
```diff
@@ -2,7 +2,7 @@
 type Cron::Month = Variant[
   Cron::Monthname,
   Integer[1,12],
-  Pattern[/(?x)\A(
+Pattern[/(?x)\A(
     \* ( \/ ( 1[012]?|[2-9] ) )?
```
Comment (igalic) [inline types/month.pp:6]: why is this indentation gone?

File: types/weekday.pp [hunk 5]
```diff
@@ -2,7 +2,7 @@
 type Cron::Weekday = Variant[
   Cron::Weekdayname,
   Integer[0,7],
-  Pattern[/(?x)\A(
+Pattern[/(?x)\A(
```
Comment (igalic) [inline types/weekday.pp:5]: why is this indentation gone?

File: types/weekdayname.pp [hunk 6]
```diff
@@ -1,3 +1,3 @@
 # Short names for each day of the week.
 type Cron::Weekdayname = Enum['Sun','Mon','Tue','Wed',
-                              'Thu','Fri','Sat']
+'Thu','Fri','Sat']
```
Comment (igalic) [inline types/weekdayname.pp:3]: if we're changing this already, it would probably be better readable either on one line, or else one day per line

**Comment by jcpunk** [comment]
The various indentation oddities are the result of `puppet-lint --relative --fix` I'm not sure how it arrived at the choices it made.

File: types/weekdayname.pp [hunk 7]
```diff
@@ -1,3 +1,2 @@
 # Short names for each day of the week.
-type Cron::Weekdayname = Enum['Sun','Mon','Tue','Wed',
-                              'Thu','Fri','Sat']
+type Cron::Weekdayname = Enum['Sun','Mon','Tue','Wed', 'Thu','Fri','Sat']
```
Comment (kenyon) [inline types/weekdayname.pp:2]: Inconsistent spacing after commas.

File: types/monthname.pp [hunk 8]
```diff
@@ -1,3 +1,3 @@
 # Short-names for each month.
 type Cron::Monthname = Enum['Jan','Feb','Mar','Apr','May','Jun',
```
Comment (bastelfreak) [inline types/monthname.pp:2]: I think we should write this into one line

**Review by igalic — APPROVED** [review]

**Pull request merged**